Firestore/core/src/firebase/firestore/timestamp.cc

```diff
@@ -31,7 +31,7 @@ namespace firebase {
 
 namespace {
 
-constexpr int32_t kNanosPerSecond = 1E9;
+constexpr int32_t kNanosPerSecond = 1000000000;
```

Comment:
Optional: consider writing this as `1*1000*1000*1000` to make counting zeroes easier.

Reply:
I did `1000 * 1000 * 1000`.